addons/account_audit_trail/models/mail_message.py

```diff
@@ -22,7 +22,7 @@ class Message(models.Model):
     def _compute_account_audit_log_preview(self):
         for message in self:
             title = message.subject or message.preview
-            tracking_value_ids = message.sudo().tracking_value_ids.filtered(lambda tracking: not tracking.field_groups or self.env.is_superuser() or self.user_has_groups(tracking.field_groups))
+            tracking_value_ids = message.sudo().tracking_value_ids.filtered(lambda tracking: not tracking.field_groups or self.env.is_superuser() or self.env.user._has_group(tracking.field_groups))
```

Comment:
`tracking.field_groups` is a comma-separated sequence of group XML ids, as this field's value is computed from some `Field`'s attribute `groups`...